### PR TITLE
Add sd mux

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
-## [Unreleased](https://github.com/3mdeb/meta-rte/compare/v0.7.3...HEAD)
+## [Unreleased](https://github.com/3mdeb/meta-rte/compare/v0.7.4-rc1...HEAD)
```
Comment (tomzy-0) [inline CHANGELOG.md]: @PLangowski please, remove this, we will add the Changelog before the release of v0.7.4

File: recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bb [hunk 2]
```diff
@@ -5,11 +5,15 @@ HOMEPAGE = "https://github.com/3mdeb/RteCtrl/blob/master/README.md"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://${S}/src/${GO_IMPORT}/LICENSE;md5=50335162666472aa33383f35cce3e355"
 
-TAG = "0.5.2"
-SRC_URI = "git://git@github.com/3mdeb/RteCtrl.git;protocol=https;branch=master;tag=${TAG}"
+TAG = "0.5.3-rc2"
```
Comment (tomzy-0) [inline recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bb]: @PLangowski let's remove this change (about rtectrl) from this pull request - we did not test it enough to add to this release.

Comment (PLangowski) [reply recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bb]: you mean remove this commit (https://github.com/3mdeb/meta-rte/commit/ef8d86a0d0651422aa0a65bdd4aeb1895145433a) from the PR?

File: CHANGELOG.md (hunk 1 above)
Comment (PLangowski) [reply CHANGELOG.md]: Same question, is it better if I drop the commit(s) that changed this or should I create new commits that will delete this?

**Pull request closed without merging**